src/lib/common/BaselayerSwitcher.svelte

```diff
@@ -21,6 +21,7 @@
   choices={[
     ["streets", "Streets"],
     ["hybrid", "Satellite"],
+    ["dataviz", "Dataviz"],
```

Comment:
MapTiler has a bunch of basemap styles. This one looks the most neutral to me, showing enough to help people orient, but getting out of the way to let the data on top show clearly